TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/wwwroot/Styles/site.scss

```diff
@@ -1,3 +1,5 @@
+
+
```

Comment:
Can we remove the `.css` file from git too?

Reply:
Already done